rubin_scheduler/utils/consdb.py

```diff
@@ -520,7 +520,41 @@ def note(self) -> pd.Series:
         note : `pd.Series`
             Note for each visit.
         """
-        return self.consdb_visits["observation_reason"]
+        if "note" in self.consdb_visits:
+            note = self.consdb_visits["note"]
+        else:
+            note = pd.Series("", index=self.consdb_visits.index)
+        return note
```

Comment:
If "note" is really "scheduler_note" we should probably start using "scheduler_note" here as well as other places to avoid confusion.

Reply:
I went ahead and added `scheduler_note` as its own thing in addition to `note`, taking its value from `scheduler_note` from `consdb` (if it exists). Once `note` is removed from the [officially documented schema](https://rubin-scheduler.lsst.io/fbs-output-schema.html), we should also remove it here.